src/content/support/overview/index.md

```diff
@@ -0,0 +1,68 @@
+---
+linkTitle: Overview
+title: Customer support
+description: The support we provide is an essential part of our offering. Here we explain various support service processes and workflows.
+weight: 10
+last_review_date: 2024-11-25
+user_questions:
+  - What should I know when working with Giant Swarm's support staff?
+  - How is Giant Swarm organizing support?
+menu:
+  principal:
+    parent: support
+    identifier: support-overview
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-planeteers
+---
+
+Giant Swarm has developed a custom support model to offer exceptional assistance by defining different layers of support.
+
+## Direct support via Slack
+
+Our first level of support involves close interactions via Slack, ensuring bi-directional feedback as quickly as possible. This channel is also used to answer a variety of questions, which can extend beyond the platform to include anything cloud-native.
+
+The first level support is available from 08:00 to 18:00 (CET) on Monday-Friday. With a distributed team across the world, questions are often answered outside these hours. Support shifts rotate across teams, focusing on channels with clear internal handovers.
+
+In case the first line support is unable to resolve your request, it is escalated to an engineer from the team responsible of the component or application in question. This is managed through a 24-hour rotating shift.
```

Comment:
```suggestion
In case the first line support is unable to resolve your request, it's escalated to an engineer from the team responsible of the component or application in question. This is managed through a 24-hour rotating shift.
```